src/queries/user/collection/articles.ts

```diff
@@ -29,20 +29,40 @@ const resolver: GQLCollectionResolvers['articles'] = async (
     return connectionFromArray([], { first, after }, count)
   }
 
-  const [articles, totalCount] =
-    await collectionService.findAndCountArticlesInCollection(collectionId, {
-      skip,
-      take,
-      reversed,
-    })
-
-  return connectionFromPromisedArray(
-    atomService.articleIdLoader.loadMany(
-      articles.map(({ articleId }) => articleId)
-    ),
-    { first, after },
-    totalCount
-  )
+  const loadArticles = async (
+    articles: CollectionArticle[],
+    totalCount: number,
+    pageNumber: number | null = 1
+  ) => {
+    const connection = await connectionFromPromisedArray(
+      atomService.articleIdLoader.loadMany(
+        articles.map(({ articleId: aid }) => aid)
+      ),
+      { first, after },
+      totalCount
+    )
+
+    return pageNumber ? { ...connection, pageNumber } : connection
```

Comment:
`pageNumber` do not exist in graphql schema?

Reply:
Thanks for pointing it out! I just pushed the changes here: https://github.com/thematters/matters-server/pull/3994/files#diff-efc1675187620595b0844197a25c35eac9b6df752f9182e5cffddee6f27a8489R3463 Let me know if this is not the right place and I can make edits